src/components/auth/registerUser.js

```diff
@@ -0,0 +1,228 @@
+import React, { Component } from "react";
+import Input from "../common/input";
+import Button from "../common/button";
+import "./../../assets/authAssets/test.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { connect } from "react-redux";
+
+import { signUpAction } from "../../actions/authActions/signUpAction";
+
+export class RegisterUser extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newUser: {
```

Comment:
No need to use a constructor here as you are not passing any props in the state. Simply use state={}

Reply:
I have removed the constructor and put the state directly into the componet